tracer/test/Datadog.Trace.Tests/Configuration/ImmutableIntegrationSettingsTests.cs

```diff
@@ -25,6 +25,7 @@ public void ImmutableSettingsRespectsOverride(bool initiallyEnabled)
             var immutableSettings = new ImmutableIntegrationSettings(settings, isExplicitlyDisabled: true);
 
             immutableSettings.Enabled.Should().BeFalse();
+            immutableSettings.ToString().Should<string>().Be($"Kafka - Disabled - Analytics: Disabled");
```

Comment:
Could you rather assert the public properties here please?

Reply:
Done